shell/pages/c/_cluster/uiplugins/InstallDialog.vue

```diff
@@ -82,25 +82,31 @@ export default {
   },
 
   methods: {
+    getChartVersion(pluginVersion) {
```

Comment:
can we add this function to https://github.com/rancher/dashboard/blob/master/shell/utils/uiplugins.ts and update it's usage on this PR?

Reply:
Done